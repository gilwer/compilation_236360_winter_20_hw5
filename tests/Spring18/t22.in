int mod(int lhs, int rhs) {
	rhs = (lhs / rhs) * rhs;
	return lhs - rhs;
}

void main() {
	print("0 mod 10 = ");
	printi(mod(0,10));
	print("16 mod 2 = ");
	printi(mod(16,2));
	print("17 mod 3 = ");
	printi(mod(17,3));
}