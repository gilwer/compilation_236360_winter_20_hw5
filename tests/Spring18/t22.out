0 mod 10 = 
0
16 mod 2 = 
0
17 mod 3 = 
2
